src/Engine/Audio/SoundBuffer.cpp

```diff
@@ -13,42 +17,71 @@
 using namespace Audio;
 
 SoundBuffer::SoundBuffer() {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Member variable 'SoundBuffer::buffer' is not initialized in the constructor. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AuninitMemberVar)